### PR TITLE
Add multi issue example to git message guidelines.

### PR DESCRIPTION
**Pull request by zjhuntin**
Just adds a small example to guidelines for this particular situation.

### CODE REVIEW

**Review by ehelms — APPROVED** [review]

File: handbook.md [hunk 1]
```diff
@@ -80,6 +80,17 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple issues you can follow this example:
+
+```
+Fixes #1234,#5678 - Fix content view yum publish bug
```
Comment (domcleal) [inline handbook.md]: Please improve this example by not repeating "Fix" and to be more specific (match the style or simply re-use the previous example).

File: handbook.md [hunk 2]
```diff
@@ -80,6 +80,17 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple issues you can follow this example:
```
Comment (domcleal) [inline handbook.md]: "multiple, distinct issues", else they should be marked as duplicates

File: handbook.md [hunk 3]
```diff
@@ -80,6 +80,17 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
...
+
+```
+Fixes #1234,#5678 - Fix content view yum publish bug
+
+Fix a problem where content views were not being published if they
```
Comment (domcleal) [inline handbook.md]: Not necessary to include this?

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by zjhuntin** [comment]
@domcleal Updated, mind taking another look?

File: handbook.md [hunk 4]
```diff
@@ -80,6 +80,16 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple, distinct issues you can follow this example:
+
+```
+Fixes #1234,#5678 - Content view yum publish bug
+
+Repositories were not being referenced correctly.
```
Comment (domcleal) [inline handbook.md]: As previous comment (https://github.com/theforeman/theforeman.org/pull/732#discussion_r87147507), why include this paragraph? It's not relevant to the example.

File: handbook.md [hunk 5]
```diff
@@ -80,6 +80,16 @@ Optionally, include more detailed explanatory text if necessary and wrap it to a
 
 Ensure the description is of the change and not the bug title, e.g. "X now accepts Y when doing Z" rather than "Z throws error".
 
+If the commit that you are adding fixes multiple, distinct issues you can follow this example:
+
+```
+Fixes #1234,#5678 - Content view yum publish bug
```
Comment (domcleal) [inline handbook.md]: Again, please use the existing example.

**Review by domcleal — CHANGES_REQUESTED** [review]
Requested changes not made.

**Comment by zjhuntin** [comment]
@domcleal I think this follows the previous example now.

**Comment by domcleal** [comment]
Merged, thanks.

**Pull request merged**